packages/lesswrong/server/resolvers/jargonResolvers/jargonTermMutations.ts

```diff
@@ -0,0 +1,256 @@
+    if (latexPattern.test(ariaLabel)) {
+      results.push(ariaLabel);
+    }
+  }
+  
+  return results.join(",");
+}
+
+
+const getMathExample = (() => {
+  let examplePostContents: string;
+
+  return async () => {
+    if (!examplePostContents) {
+      const pathName = './public/exampleMathPost.md'
+      examplePostContents = (await readFile(pathName)).toString()
+    }
+
+    return examplePostContents;
+  };
+})();
+
+
+export async function getLaTeXExplanations(post: PostsPage) {
+  const originalHtml = post.contents?.html ?? ""
+  const originalMarkdown = htmlToMarkdown(originalHtml)
+  const markdown = (originalMarkdown.length < 200 * 1000) ? originalMarkdown : originalMarkdown.slice(0, 200 * 1000)
+
+  const terms = identifyLatexTerms(originalHtml)
+  const exampleMathPost = await getMathExample();
+
+  const response = (await queryClaudeForJargonExplanations({markdown, terms, formatPrompt: mathFormatPrompt, examplePost: exampleMathPost, exampleExplanations: exampleMathGlossary})) || []
+
+  return response?.map(jargon => ({
+    term: jargon.term,
+    text: `<div>
+      <p>${jargon.text}</p>
+      <p><strong>Concrete Example:</strong> ${jargon.concreteExample}</p>
+      <p><strong>Why It Matters:</strong> ${jargon.whyItMatters}</p>
+      <p><em>Math Basics: ${jargon.mathBasics}</em></p>
+    </div>`,
+    altTerms: jargon.altTerms,
+  }))
+}
+
+export async function createEnglishExplanations(post: PostsPage) {
+  const originalHtml = post.contents?.html ?? ""
+  const originalMarkdown = htmlToMarkdown(originalHtml)
+  const markdown = (originalMarkdown.length < 200 * 1000) ? originalMarkdown : originalMarkdown.slice(0, 200 * 1000)
+
+  const terms = await queryClaudeForTerms(markdown)
+  if (!terms) return []
+
+  return queryClaudeForJargonExplanations({markdown, terms, formatPrompt, examplePost: exampleJargonPost2, exampleExplanations: exampleJargonGlossary2})
+}
+
+export const createNewJargonTerms = async (postId: string, currentUser: DbUser) => {
+  const post = await fetchFragmentSingle({
+    collectionName: 'Posts',
+    fragmentName: 'PostsPage',
+    currentUser,
+    selector: { _id: postId },
+  });
+  if (!post) {
+    throw new Error('Post not found');
+  }
+  // console.log("creatingLatexTerms")
+  // const newMathJargon = await getLaTeXExplanations(post) || []
+  const newEnglishJargon = await createEnglishExplanations(post) || []
+
+  console.log("creating jargonTerms")
+  const newJargonTerms = await Promise.all([
+    ...newEnglishJargon.map(term =>
+      createMutator({
+        collection: JargonTerms,
+        document: {
+          postId: postId,
+          term: term.term,
+          humansAndOrAIEdited: "AI",
+          forLaTeX: false,
+          rejected: true,
+          contents: {
+            originalContents: {
+              data: term.text,
+              type: 'ckEditorMarkup',
+            },
+          },
+          altTerms: term.altTerms,
+        },
+        currentUser: currentUser,
```

Comment:
This should probably be using a dedicated bot/admin account so we can derive the `humansAndOrAIEdited` value from the revision history (see relevant PR comment on the schema file)